pkg/controller/backup_deletion_controller.go

```diff
@@ -102,7 +102,7 @@ func NewBackupDeletionReconciler(
 
 func (r *backupDeletionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	// Make sure the expired requests can be deleted eventually
-	s := kube.NewPeriodicalEnqueueSource(r.logger, mgr.GetClient(), &velerov1api.DeleteBackupRequestList{}, time.Hour, kube.PeriodicalEnqueueSourceOption{})
+	s := kube.NewPeriodicalEnqueueSource("backupDeletion", r.logger, mgr.GetClient(), &velerov1api.DeleteBackupRequestList{}, time.Hour, kube.PeriodicalEnqueueSourceOption{})
```

Comment:
Use the constant defined here https://github.com/vmware-tanzu/velero/blob/main/pkg/constant/constant.go

Reply:
You can add the controller information like this:
```go
kube.NewPeriodicalEnqueueSource(r.logger.WithField("controller", controllerName))
```
In this way we could add any necessary information without modifying the definition of `NewPeriodicalEnqueueSource()` everytime

Reply:
Will update. Thanks!